### PR TITLE
Remove logging to text files without python logger

### PR DESCRIPTION
**Pull request by Dominik-Vogel**

### CODE REVIEW

File: qdev_wrappers/templates/initialisation_script.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 from qdev_wrappers import logging
 
 # import modules you might want to use
-import atexit
```
[the remark this answers is not in the document]
Comment (ThorvaldLarsen) [reply qdev_wrappers/templates/initialisation_script.py]: Great! Thanks for pointting that out.

**Comment by ThorvaldLarsen** [comment]
@Dominik-Vogel this part has been running perfectly fine on T11 so think we can merge it. However, there are some conflicts that needs to be resolved before I can merge.

**Comment by ThorvaldLarsen** [comment]
Actually I guess this PR assumes #76 is merged...

**Comment by jenshnielsen** [comment]
I suggest that we rebase this before merging. It touches files that are no longer there. I am a bit confused why that is not a conflict

**Comment by jenshnielsen** [comment]
@ThorvaldLarsen Yes I agree, that was what I was trying to say

**Pull request merged**